pyairtable/api/abstract.py

```diff
@@ -16,25 +16,28 @@
 
 class ApiAbstract(metaclass=abc.ABCMeta):
     VERSION = "v0"
-    API_BASE_URL = "https://api.airtable.com/"
     API_LIMIT = 1.0 / 5  # 5 per second
-    API_URL = posixpath.join(API_BASE_URL, VERSION)
     MAX_RECORDS_PER_REQUEST = 10
 
     session: Session
+    endpoint_url: str
+    api_url: str
     tiemout: TimeoutTuple
 
     def __init__(
         self,
         api_key: str,
         timeout: Optional[TimeoutTuple] = None,
         retry_strategy: Optional[Retry] = None,
+        endpoint_url: Optional[str] = "https://api.airtable.com",
```

Comment:
Since `None` is not a valid value, I think this should be `str` instead of `Optional[str]`.

Reply:
https://github.com/gtalarico/pyairtable/pull/243/commits/f1937d7b73e8c0f7cd339adffb0f58bfea8b33a7